fatiando/gravmag/transform.py

```diff
@@ -118,6 +125,106 @@ def tga(x, y, data, shape, method='fd'):
     return res
 
 
+def mstde(x, y, data, shape):
+    """
+    Multi-Scale Tilt Depth Estimation
+
+    Implements the method of van Buren (2013).
+
+    An empirical method for attempting to approximate the depth of the source
+    of a magnetic anomaly, based on altering tilt and continuation of the
+    magnetic field intensity.
+
+    This method will also work well on Total Gradient Amplitude, but care must
+    be taken to reduce noise.
+
+    .. warning::
+
+        This algorithm expects Reduced to Pole data. Within `fatiando`, the
+        recommended method is to use equivalent layers to achieve this. See
+        `cookbook/gravmag_eqlayer_pel_polereduc.py` and
+        `cookbook/gravmag_eqlayer_polereduc.py` for an idea of how this can be
+        done.
+
+    **References**
+
+    Van Buren, Reece. 2013."Multi-Scale Tilt Depth Estimation." MSc Thesis,
+    Johannesberg: University of the Witwatersrand.
+    URI: http://mobile.wiredspace.wits.ac.za/bitstream/handle/10539/14011/
+    MSc_Dissertation_RvanBuren_2013.pdf.
+    """
+    RTP_TDR = tilt(x, y, data, shape)
+    AREA = numpy.arctan2( numpy.tan( (numpy.pi/2)-numpy.arctan2(RTP_TDR) ) )
```

Comment:
Has this been published yet? If not, could you point me to the page/equation in the thesis that you're implementing here?